core/src/libraries/constants/Constants.sol

```diff
@@ -10,15 +10,17 @@ uint256 constant BORROWS_SCALER = type(uint72).max * ONE;
 
 /// @dev The maximum percentage yield per second, scaled up by 1e12. The current value is equivalent to
 /// (1 + 706354 / 1e12) ^ (24 * 60 * 60) - 1 ⇒ +6.3% per day or +53% per week. If the rate is consistently at this
```

Comment:
Can we put this in TeX? 🥺

Reply:
I changed it to Python-style in-line code. Hopefully that's ok!